lib/Doctrine/Collection/Iterator/Offset.php

```diff
@@ -32,6 +32,7 @@
  */
 class Doctrine_Collection_Iterator_Offset extends Doctrine_Collection_Iterator
 {
+    #[\ReturnTypeWillChange]
     public function valid()
     { }
 }
```

Comment:
new line at the end is missing?

Reply:
They are missing in many files. I think gh just shows it here, because the diff includes the end of the file.

Should be solved in another pr fixing php-cs issues?

Reply:
sorry, I got confused why php-cs-fixer did not fix it

The PR #112 is not merged yet :(